src/client/dfuse/SConscript

```diff
@@ -125,20 +125,19 @@ def check_struct_member(context, text, struct, member):
 def check_ioctl_def(context, ctype):
     """scons check for a struct member existing"""
 
-    context.Message(f'Checking if fuse ioctl is type {ctype} ')
-
-    src = """#include <fuse3/fuse_lowlevel.h>
+    def _src():
 
-extern void
-my_ioctl (fuse_req_t req, fuse_ino_t ino, %s cmd,
-    void *arg, struct fuse_file_info *fi, unsigned flags,
-    const void *in_buf, size_t in_bufsz, size_t out_bufsz);
+        return (
+            '#include <fuse3/fuse_lowlevel.h>\n'
+            f'extern void my_ioctl (fuse_req_t req, fuse_ino_t ino, {ctype} cmd, void *arg,\n'
+            'struct fuse_file_info *fi, unsigned flags, const void *in_buf, size_t in_bufsz,\n'
+            'size_t out_bufsz);'
+            'struct fuse_lowlevel_ops ops = {.ioctl = my_ioctl};'
+        )
 
-struct fuse_lowlevel_ops ops = {.ioctl = my_ioctl};
-
-""" % ctype
+    context.Message(f'Checking if fuse ioctl is type {ctype} ')
 
-    rc = context.TryCompile(src, '.c')
+    rc = context.TryCompile(_src(), '.c')
```

Comment:
(pylint-invalid-name) Variable name "rc" doesn't conform to snake_case naming style